### PR TITLE
Microsoft defender for cloud

### PR DESCRIPTION
**Pull request by JV0812**
## Purpose of this pull request

This pull request Microsoft defender for Cloud apps doc.

## Select the type of change

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions, updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

## Ticket (if applicable)

[Create doc for Microsoft Defender for Cloud App](https://sumologic.atlassian.net/browse/DOCS-561)

### CODE REVIEW

File: sidebars.ts [hunk 1]
```diff
@@ -2142,6 +2142,7 @@ integrations: [
          'integrations/microsoft-azure/azure-synapse-analytics',
          'integrations/microsoft-azure/azure-virtual-network',
          'integrations/microsoft-azure/kubernetes',
+         `integrations/microsoft-azure/microsoft-defender-for-cloud',
```
Comment (kimsauce) [inline sidebars.ts:2145]:
```suggestion
         'integrations/microsoft-azure/microsoft-defender-for-cloud',
```

Comment (kimsauce) [reply sidebars.ts:2145]: This will fix the build error

File: blog-service/2024-12-02-apps.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  - microsoft-defender-for-cloud
+  - apps
+hide_table_of_contents: true
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<a href="https://help.sumologic.com/release-notes-service/rss.xml"><img src={useBaseUrl('img/release-notes/rss-orange2.png')} alt="icon" width="50"/></a>
+
+We're excited to introduce the new Microsoft Defender for Cloud app for Sumo Logic. This app helps you to collect the alerts, security recommendation, and regulatory compliance logs using the Sumo Logic Cloud-to-Cloud Azure Event Hub Source and by configuring the continuos export using the Event Hub instance details in the Azure portal. Key features of the Microsoft Defender for Cloud app include:
```
Comment (jpipkin1) [inline blog-service/2024-12-02-apps.md:14]:
```suggestion
We're excited to introduce the new Microsoft Defender for Cloud app for Sumo Logic. This app helps you to collect the alerts, security recommendation, and regulatory compliance logs using the Sumo Logic Cloud-to-Cloud Azure Event Hub Source and by configuring the continuous export using the Event Hub instance details in the Azure portal. Key features of the Microsoft Defender for Cloud app include:
```

File: docs/integrations/microsoft-azure/index.md [hunk 3]
```diff
@@ -317,6 +317,12 @@ This guide has documentation for all of the apps that Sumo Logic provides for Mi
       <p>A guide to the Sumo Logic app for Azure Kubernetes Service Control Plane.</p>
       </div>
     </div>
+<div className="box smallbox card">
+  <div className="container">
+  <a href="/docs/integrations/security-threat-detection/microsoft-defender-for-cloud"><img src={useBaseUrl('img/integrations/microsoft-azure/microsoft-defender-for-cloud.png')} alt="thumbnail icon" width="75"/><h4>Microsoft Defender for Cloud</h4></a>
```
Comment (jpipkin1) [inline docs/integrations/microsoft-azure/index.md:322]:
```suggestion
  <a href="/docs/integrations/microsoft-azure/microsoft-defender-for-cloud"><img src={useBaseUrl('img/integrations/microsoft-azure/microsoft-defender-for-cloud.png')} alt="thumbnail icon" width="75"/><h4>Microsoft Defender for Cloud</h4></a>
```

File: docs/integrations/microsoft-azure/microsoft-defender-for-cloud.md [hunk 4]
```diff
@@ -0,0 +1,399 @@
...
+
+## Troubleshooting
+
+### Verify Event Hub data flow
+
+If your configured Event Hub instance is not successfully sending data to Sumo Logic. Follow the below steps to troubleshoot the issue:
+
+1. Navigate to the **Event Hub Instance Blade** in the Azure portal and select the **Data Explorer(preview)** tab to send sample events..<br/> <img src={useBaseUrl('img/integrations/microsoft-azure/event-hub-instance-blade.png')} style={{border:'1px solid gray'}} alt="event-hub-instance-blade" width="800"/>
+1. In the **Data Explorer(preview)** page, click **Send event** and preview the sample events.<br/> <img src={useBaseUrl('img/integrations/microsoft-azure/data-explorer.png')} style={{border:'1px solid gray'}} alt="data-explorer" width="800"/>
+1. Verify if those events are being sent to the [Sumo Logic by Live Tailing](/docs/search/live-tail/about-live-tail/). If both the data matches, then event hub instance will be successfully sending data to Sumo Logic. <br/> <img src={useBaseUrl('img/integrations/microsoft-azure/live-tailing.png')} style={{border:'1px solid gray'}} alt="live-tailing" width="800"/>
```
Comment (jpipkin1) [inline docs/integrations/microsoft-azure/microsoft-defender-for-cloud.md:373]:
```suggestion
1. Navigate to the **Event Hub Instance Blade** in the Azure portal and select the **Data Explorer (preview)** tab to send sample events.<br/> <img src={useBaseUrl('img/integrations/microsoft-azure/event-hub-instance-blade.png')} style={{border:'1px solid gray'}} alt="event-hub-instance-blade" width="800"/>
1. In the **Data Explorer (preview)** page, click **Send event** and preview the sample events.<br/> <img src={useBaseUrl('img/integrations/microsoft-azure/data-explorer.png')} style={{border:'1px solid gray'}} alt="data-explorer" width="800"/>
1. Verify if those events are being sent to the [Sumo Logic by Live Tailing](/docs/search/live-tail/about-live-tail/). If the data matches, then the Event Hub instance will successfully send data to Sumo Logic. <br/> <img src={useBaseUrl('img/integrations/microsoft-azure/live-tailing.png')} style={{border:'1px solid gray'}} alt="live-tailing" width="800"/>
```

File: docs/integrations/microsoft-azure/microsoft-defender-for-cloud.md [hunk 5]
```diff
@@ -0,0 +1,399 @@
...
+
+### Validate alerts at Event Hub
+
+If you are not receiving any alerts from the Microsoft Defender to the Event Hub instance. Firstly, make sure that the generated sample alerts are received in your configured Event Hub instance. This ensures the connection between Defender and Event Hub is functioning correctly. To test the pipeline by sending sample alerts from Microsoft Defender by following the below steps:
```
Comment (jpipkin1) [inline docs/integrations/microsoft-azure/microsoft-defender-for-cloud.md:377]:
```suggestion
If you are not receiving any alerts from the Microsoft Defender to the Event Hub instance, first make sure that the generated sample alerts are received in your configured Event Hub instance. This ensures the connection between Defender and Event Hub is functioning correctly. To test the pipeline by sending sample alerts from Microsoft Defender, follow the below steps:
```

**Review by jpipkin1 — APPROVED** [review]
Very nice work, @JV0812! I have only a few minor suggestions.

**Comment by kimsauce** [comment]
Publishing per @ankitgoelcmu

**Pull request merged**